crypto/s2n_sequence.h

```diff
@@ -20,3 +20,4 @@
 #include "utils/s2n_blob.h"
 
 extern int s2n_increment_sequence_number(struct s2n_blob *sequence_number);
+int s2n_sequence_number_to_uint64(struct s2n_blob *sequence_number, uint64_t *output);
```

Comment:
Are we guaranteed that a sequence number will always fit in `uint64`?

Reply:
The max value of a sequence number is 2^64, so yes, it should always fit in a uint64. If it does go above 2^64, we close the connection.